### DISCUSSION TITLE
Kitty theme switching, avoiding edits to kitty.conf

### DISCUSSION THREAD

**Discussion by ldelossa** [kovidgoyal/kitty#4579] [Q&A] [answered]
Hello, 

I often do a theme switch on my laptop when moving from outdoors to indoors. 

I also check in my dotfiles to git. Normally how this works is I have two config files "app-dark.conf" and "app-light.conf" and I have a script which symlinks "app.conf" to either the dark or light configuration files. And then I gitignore "app.conf" so theme switching doesn't create a diff. 

When I theme switch with Kitty, it edits "kitty.conf" and causes a diff in my dotfiles repository. Is there anyway to avoid this? Maybe by having kitty reference a different file for theme and gitignoring this?

**Comment by kovidgoyal** [comment, marked as answer]
Have kitty.conf use include to include the real config and gitignore kitty.conf.

**Reply by ldelossa** [reply]
Thanks, very simple, dunno why i didn't see it. Thanks for being the rubber duck lol.
